lang/src/org/partiql/lang/eval/NaturalExprValueComparators.kt

```diff
@@ -104,8 +115,10 @@ enum class NaturalExprValueComparators(private val nullOrder: NullOrder) : Compa
         val rIter = right.iterator()
 
         while (lIter.hasNext() && rIter.hasNext()) {
-            val lChild = lIter.next()
-            val rChild = rIter.next()
+            val (lChild, rChild) = when(order) {
```

Comment:
Seems like GitHub actions builds aren't run for forks as mentioned in #561. When I build locally, this line caused a ktlint error. Fixable by running `./gradlew ktlintFormat` or applying the following suggestion:

```suggestion
            val (lChild, rChild) = when (order) {
```

Reply:
Similarly with L142.